examples/Isentropic_Interpolation.py

```diff
@@ -0,0 +1,149 @@
+now = datetime.utcnow()
+start = datetime(2017, 7, 10, 12)
+
+# define time range you want the data for
+end = start + timedelta(hours=12)
+
+gfsdata = ncss.query().time_range(start, end)
+
+gfsdata.variables('Temperature_isobaric',
+                  'u-component_of_wind_isobaric',
+                  'v-component_of_wind_isobaric',
+                  'Relative_humidity_isobaric').add_lonlat()
+
+# Set the lat/lon box for the data you want to pull in.
+# lonlat_box(north_lat,south_lat,east_lon,west_lon)
+gfsdata.lonlat_box(-150, -50, 15, 65)
+
+# Actually getting the data
+data = ncss.get_data(gfsdata)
+
+dtime = data.variables['Temperature_isobaric'].dimensions[0]
+dlev = data.variables['Temperature_isobaric'].dimensions[1]
+lat = data.variables['lat'][:]
+lon = data.variables['lon'][:]
+lev = data.variables[dlev][:] * units.Pa
+times = data.variables[dtime]
+vtimes = num2date(times[:], times.units)
+temps = data.variables['Temperature_isobaric']
+tmp = temps[:] * units.kelvin
+uwnd = data.variables['u-component_of_wind_isobaric'][:] * units.meter / units.second
+vwnd = data.variables['v-component_of_wind_isobaric'][:] * units.meter / units.second
+relh = data.variables['Relative_humidity_isobaric'][:]
+
+########################
+# To properly interpolate to isentropic coordinates, the function must know the desired output
+# isentropic levels. An array with these levels will be created below.
+
+isentlevs = np.arange(310, 316, 5) * units.kelvin
+
+########################
+# Conversion to Isentropic Coordinates
+#
+# Once model data in isobaric coordinates has been pulled and the desired isentropic levels
+# created, the conversion to isentropic coordinates can begin. Data will be passed to the
+# function as below. The function requires that isentropic levels, isobaric levels, and
+# temperature be input. Any additional inputs (in this case relative humidity, u, and v wind
+# components) will be linearly interpolated to isentropic space.
+
+isent_anal = metpy.calc.isentropic_interpolation(isentlevs, lev, tmp, relh, uwnd, vwnd, axis=1)
+
+####################
+# The output is a tuple, so now we will separate the variables to different names before
+# plotting.
+
+isentprs = isent_anal[0]
+isentrh = isent_anal[1]
+isentu = isent_anal[2]
+isentv = isent_anal[3]
+
+##########################
+# A quick look at the shape of these variables will show that the data is now in isentropic
+# coordinates, with the number of vertical levels as specified above.
+
+print(isentprs.shape)
+print(isentrh.shape)
+print(isentu.shape)
+print(isentv.shape)
+
+####################
+# **Plotting the Isentropic Analysis**
+
+#####################
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+clons, clats = np.meshgrid(lon, lat)
+tlatlons = crs.transform_points(ccrs.PlateCarree(), clons, clats)
```

Comment:
Why transform manually? Instead you should be able to pass `transform=ccrs.PlateCarree()` to `contour` and friends.